topostats/filters.py

```diff
@@ -35,6 +36,7 @@ def __init__(
         threshold_absolute: dict = None,
         gaussian_size: float = None,
         gaussian_mode: str = "nearest",
+        remove_scars: dict = None,
```

Comment:
@ns-rse It passes for me 🤔 

```
❯ git checkout SylviaWhittle/remove_scars
Already on 'SylviaWhittle/remove_scars'
Your branch is up to date with 'origin/SylviaWhittle/remove_scars'.
❯ pylint topostats/filters.py

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

Are we sure that it's the `dict = None` line? When I asked you about the issue, it was in relation to `plottingfuncs.py` and not `scars.py`, and it has a few other dictionary arguments?

My pylint setup is behaving very funky though. It flags some imports (that **definitely** work when running the code) as unable to be imported. Eg:

```
❯ pylint tests/test_scars.py
************* Module test_scars
tests/test_scars.py:6:0: E0401: Unable to import 'topostats' (import-error)

------------------------------------------------------------------
Your code has been rated at 8.65/10 (previous run: 8.65/10, -0.00)
```

So maybe it's something broken with my pylint?

Reply:
Ah, my confusion about which file you'd asked me about, sorry.

I've seen that sort of error before and found it confusing too. It might be a [VSCode/venv/pylint](https://stackoverflow.com/questions/51095449/pylint-false-positive-e0401-import-errors-in-vscode-while-using-venv) conflict that is raising it.